src/Components/Common/Table/Table.module.scss

```diff
@@ -0,0 +1,46 @@
+.tableContainer {
+  width: 750px;
+  padding: $indentInputAndButton $indentMin 0;
+  box-sizing: content-box;
+
+  .table {
+    width: 100%;
+    border-collapse: collapse;
+    border: 1px solid $content-text;
+    font-size: 0.7rem;
+
+    .tr {
+      border: 1px solid $content-text;
+
+      .th,
+      .td {
+        border: 1px solid $content-text;
+      }
+
+      .th:first-child {
+        width: 15%;
+      }
+
+      .td {
+        text-align: center;
+      }
+    }
+  }
+
+  @include respond($laptop) {
+    width: 700px;
+  }
+  @include respond($tabletMin) {
+    width: 100%;
+    padding: $indentInputAndButton 0 0;
+    box-sizing: border-box;
+    .table {
+      font-size: 0.55rem;
+    }
+  }
+  @include respond($mobile) {
+    .table {
+      font-size: 0.6rem;
```

Comment:
BTW, do you have constants for font sizes? I would recommend to create them and have limited set of sizes so you could easily refer to them in scss file

Reply:
Done!